replayer/src/main/java/dk/dbc/pgqueue/replayer/Database.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2020 DBC A/S (http://dbc.dk/)
+ *
+ * This is part of pg-queue-replayer
+ *
+ * pg-queue-replayer is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * pg-queue-replayer is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package dk.dbc.pgqueue.replayer;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.sql.DataSource;
+import org.postgresql.ds.PGSimpleDataSource;
+
+/**
+ *
+ * @author Morten Bøgeskov (mb@dbc.dk)
+ */
+public class Database {
+
+    private static final Pattern POSTGRES_URL_REGEX = Pattern.compile("(?:postgres(?:ql)?://)?(?:([^:@]+)(?::([^@]*))@)?([^:/]+)(?::([1-9][0-9]*))?/(.+)");
+
+    /**
+     * Convert a login string into a datasource
+     *
+     * @param url login string
+     * @return datasource (not validated)
+     */
+    public static DataSource of(String url) {
```

Comment:
Understøtter det her kode en fuld postgres url? For, ellers synes jeg måske nok det skal fremgå af dokumentationen.

Reply:
Yep